app/models/service_ansible_playbook.rb

```diff
@@ -83,14 +83,22 @@ def get_job_options(action)
   end
 
   def save_job_options(action, overrides)
-    job_options = options.fetch_path(:config_info, action.downcase.to_sym).slice(:hosts, :extra_vars).with_indifferent_access
+    option_keys = %i[
```

Comment:
Should this whitelist be in a constant?

Reply:
Yeah, probably. I think I'll refactor some of this into methods.